packages/js/onboarding/src/components/WooOnboardingTaskListHeader/WooOnboardingTaskListHeader.tsx

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import { createElement } from '@wordpress/element';
+import { Slot, Fill } from '@wordpress/components';
+
+type WooOnboardingTaskListHeaderProps = {
+	id: string;
+};
+
+/**
+ * A Fill for adding Onboarding Task List items.
+ *
+ * @slotFill WooOnboardingTaskListHeader
+ * @scope woocommerce-tasks
+ * @param {Object} props    React props.
+ * @param {string} props.id Task id.
+ */
+export const WooOnboardingTaskListHeader = ( {
+	id,
+	...props
+}: WooOnboardingTaskListHeaderProps & Slot.Props ) => (
+	<Fill
+		name={ 'woocommerce_onboarding_task_list_header_' + id }
```

Comment:
I did not add `experimental` to this since it's similar to existing pattern: https://github.com/woocommerce/woocommerce/blob/def14e2aec83153e73a5d1b698e3d10a3ce9cdf5/packages/js/onboarding/src/components/WooOnboardingTaskListItem/WooOnboardingTaskListItem.tsx#L19-L23